src/main/java/org/purescript/parser/PSElements.kt

```diff
@@ -196,6 +196,9 @@ interface PSElements {
         @kotlin.jvm.JvmField
         val NewtypeDeclaration = PSElementType("NewtypeDeclaration")
 
+        @kotlin.jvm.JvmField
+        val NewTypeConstructor = PSElementType("NewTypeConstructor")
```

Comment:
This one does not exist in the compiler; it lets newtype declarations have two proper names. We need to split them up though, since they are two different name owners.